builder/dockerfile/builder.go

```diff
@@ -6,6 +6,7 @@ import (
 	"io"
 	"io/ioutil"
 	"runtime"
+   "strconv"
```

Comment:
Looks like the format is not properly aligned. Maybe you haven't run `gofmt -s -w`?

Reply:
I did...no errors seen :)

Reply:
something is wrong then - this formatting needs to be fixed